daml-lf/transaction/src/test/scala/com/digitalasset/daml/lf/transaction/TransactionVersionSpec.scala

```diff
@@ -151,33 +91,34 @@ class TransactionVersionSpec extends WordSpec with Matchers with TableDrivenProp
     }
   }
 
-  private[this] def assignValueVersions[Nid, Cid <: ContractId](
-      t: GenTransaction[Nid, Cid, Value[Cid]],
-  ): GenTransaction[Nid, Cid, VersionedValue[Cid]] =
-    t map3 (identity, identity, ValueVersions.assertAsVersionedValue(_, supportedValVersions))
-
-  private[this] def assignTxVersion[Nid, Cid <: ContractId](
-      t: GenTransaction[Nid, Cid, VersionedValue[Cid]],
-  ): Either[String, TransactionVersion] =
-    TransactionVersions.assignVersion(t, supportedTxVersions)
-
 }
 
+// FIXME (Remy): use or drop
```

Comment:
On what does it depend whether we use or drop this?

Reply:
I have to fix that before merging. 
Thanks for the reminder.